### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by siarheishut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
 | Field         | Type   | Description                                      |  
 |---------------|--------|--------------------------------------------------|  
 | request_id    | int    | An ID of the corresponding request.              |  
-| error_message | string | An error message, which is equal to "not found". |  
\ No newline at end of file
+| error_message | string | An error message, which is equal to "not found". |  
+
+3. Response to Route Request.
+
+- `If there isn't any route between such two stops:`
```
Comment (foxfrikses) [inline README.md:141]: `If there is no route`

File: README.md [hunk 2]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+
+- `If exists at least one route between such two stops:`
+
+| Field      | Type  | Description                                                                               |  
+|------------|-------|-------------------------------------------------------------------------------------------|  
+| request_id | int   | An ID of the corresponding request.                                                       |  
+| total_time | float | The minimum possible time to pass the route from stop `from` to stop `to`.                | 
+| items      | array | List of route elements, each of which describes the continuous activity of the passenger. | 
+
+There are 2 types of route elements:
```
Comment (foxfrikses) [inline README.md:156]: `There are 2 types of items:`

File: README.md [hunk 3]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+
+| Fields | Type | Description                                       |
+|--------|------|---------------------------------------------------| 
+| Wait   | map  | Wait `bus_wait_time` minutes on the current stop. |
+| Bus    | map  | Take a bus on the current stop.                   |
+
```
Comment (foxfrikses) [inline README.md:162]: I'd rather add the descriptions to the `Wait:` and `Bus:` lines. And `Wait` and `Bus` are not fields, nor they have type =)

File: README.md [hunk 4]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+Wait:
+
+| Fields    | Type   | Description                                      |
+|-----------|--------|--------------------------------------------------|
+| type      | string | Equal to "Wait".                                 |
+| stop_name | string | The stop where you should wait.                  |
+| time      | int    | Time you should wait (equal to `bus_wait_time`). |
```
Comment (foxfrikses) [inline README.md:169]: `The waiting time.`

File: README.md [hunk 5]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+
+Bus:
+
+| Fields     | Type   | Description                |
+|------------|--------|----------------------------|
+| type       | string | Equal to "Bus".            |
+| bus        | string | The bus you need to go on. |
```
Comment (foxfrikses) [inline README.md:176]: `The bus to get on.`

File: README.md [hunk 6]
```diff
@@ -1,16 +1,29 @@
...
+
+#### routing_settings:
+
+| Field         | Type  | Optional | Description                                            |  
+|---------------|-------|----------|--------------------------------------------------------|  
+| bus_wait_time | int   | No       | Waiting time for the bus at the bus stop (in minutes). |           
+| bus_velocity  | float | No       | The bus speed (in kilometers per hour).                |   
+
+> It is believed that whenever a person arrives at a stop and whatever that stop is,
+> he will wait for any bus for exactly `bus_wait_time` minutes.
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply README.md]: Done.

File: README.md [hunk 7]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+|---------------|--------|--------------------------------------------------| 
+| request_id    | int    | An ID of the corresponding request.              | 
+| error_message | string | An error message, which is equal to "not found". | 
+
+- `If exists at least one route between such two stops:`
+
+| Field      | Type  | Description                                                                               |  
+|------------|-------|-------------------------------------------------------------------------------------------|  
+| request_id | int   | An ID of the corresponding request.                                                       |  
+| total_time | float | The minimum possible time to pass the route from stop `from` to stop `to`.                |
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (siarheishut) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (siarheishut) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (siarheishut) [reply README.md]: Done.

File: README.md [hunk 8]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+|--------|------|---------------------------------------------------| 
+| Wait   | map  | Wait `bus_wait_time` minutes on the current stop. |
+| Bus    | map  | Take a bus on the current stop.                   |
+
+Wait:
+
+| Fields    | Type   | Description                                      |
+|-----------|--------|--------------------------------------------------|
+| type      | string | Equal to "Wait".                                 |
+| stop_name | string | The stop where you should wait.                  |
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply README.md]: Done.

File: README.md [hunk 9]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+| time      | int    | Time you should wait (equal to `bus_wait_time`). |
+
+Bus:
+
+| Fields     | Type   | Description                |
+|------------|--------|----------------------------|
+| type       | string | Equal to "Bus".            |
+| bus        | string | The bus you need to go on. |
+| span_count | int    | The count of stops to go.  |
+| time       | double | Time you have to travel.   |
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply README.md]: Done.

File: README.md [hunk 10]
```diff
@@ -112,4 +134,45 @@ a JSON map of one of the following types:
...
+| stop_name | string | The stop where you should wait.                  |
+| time      | int    | Time you should wait (equal to `bus_wait_time`). |
+
+Bus:
+
+| Fields     | Type   | Description                |
+|------------|--------|----------------------------|
+| type       | string | Equal to "Bus".            |
+| bus        | string | The bus you need to go on. |
+| span_count | int    | The count of stops to go.  |
```
[the remark this answers is not in the document]
Comment (siarheishut) [reply README.md]: Done.

**Pull request merged**